doc/taoMediaManagerApi.yml

```
@@ -38,6 +38,29 @@ paths:
           $ref: '#/components/responses/BadRequestResponse'
         500:
           $ref: '#/components/responses/InternalServerErrorResponse'
+  /taoMediaManager/SharedStimulus/patch:
+    patch:
+      summary: Updates a shared stimulus body
+      parameters:
+        - in: query
+          name: id
+          required: true
+          schema:
+            type: string
+          description: The shared stimulus id
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/SharedStimulusUpdate'
+      responses:
+        200:
+          $ref: '#/components/responses/SharedStimulusUpdatedResponse'
```

```suggestion
          $ref: '#/components/responses/SharedStimulusPatchResponse'
```

Sorry, I forgot to mention this too.